index.js

```diff
@@ -54,36 +70,48 @@ module.exports = {
       }
       if (!initSpec(content)) return cb(new Error(initSpec.errorsString))
 
-      const recipientKeys = [
-        { key: groupKey.toBuffer(), scheme: keySchemes.private_group },
-      ]
-
-      ssb.db.create(
-        {
-          content,
-          recps: recipientKeys,
-          encryptionFormat: 'box2',
-        },
-        (err, groupInitMsg) => {
-          if (err) return cb(err)
+      const groupKey = new SecretKey()
 
-          const data = {
-            id: buildGroupId({ groupInitMsg, groupKey: groupKey.toBuffer() }),
-            secret: groupKey.toBuffer(),
-            root: groupInitMsg.key,
-            subfeed: '',
-          }
+      findOrCreateGroupFeed(groupKey, (err, groupFeed) => {
+        if (err) return cb(err)
 
-          ssb.box2.addGroupInfo(data.id, { key: data.secret, root: data.root })
+        const recps = [
+          { key: groupKey.toBuffer(), scheme: keySchemes.private_group },
+        ]
 
-          // adding myself for recovery reasons
-          addMembers(data.id, [ssb.id], {}, (err) => {
+        ssb.db.create(
+          {
+            keys: groupFeed.keys,
+            content,
+            recps,
+            encryptionFormat: 'box2',
+          },
+          (err, groupInitMsg) => {
             if (err) return cb(err)
 
-            return cb(null, data)
-          })
-        }
-      )
+            const data = {
+              id: buildGroupId({ groupInitMsg, groupKey: groupKey.toBuffer() }),
+              secret: groupKey.toBuffer(),
+              root: groupInitMsg.key,
+              subfeed: groupFeed.keys,
+            }
+
+            ssb.box2.addGroupInfo(data.id, {
+              key: data.secret,
+              root: data.root,
+            })
+
+            // Adding myself for recovery reasons
+            addMembers(data.id, [ssb.id], {}, (err) => {
```

Comment:
:fire: you are adding the wrong "person" 
addMembers needs to take rootFeedIds, and this is then translated encryptionKey for the `invitations` leaf (this is the spec I need to finish writing), and published in the in the `invitations` feed.

I think we need to specify in the README which keys should go into addMembers!

Reply:
yeah so `recps` in the addMembers message should be the root ids but the actual encryption shouldn't look at those keys like it usually does :dizzy_face:

Reply:
The emoji :dizzy_face: describes me well these days

Reply:
```suggestion
            addMembers(data.id, [rootMF.keys.id], {}, (err) => {
```